polygon/deployments/outputs/137-staging.json

```diff
@@ -16,7 +16,8 @@
     "DelegatedManagerFactory": "0xa5351d5F6fE586A7A2eE0A5B7f797F5Fb686082B",
     "IssuanceExtension": "0x802569a5AC139650956f79bB0129C767cC491DE4",
     "StreamingFeeSplitExtension": "0x171Bf119b7Da5E24Ea1f41c81e0DAD41E57FeE48",
-    "TradeExtension": "0x34548881b83dC247Ca11ed4cfbe2cbeB5a2Bf0D1"
+    "TradeExtension": "0x34548881b83dC247Ca11ed4cfbe2cbeB5a2Bf0D1",
+    "TEST": "0x34548881b83dC247Ca11ed4cfbe2cbeB5a2Bf0D1"
```

Comment:
Do we want to keep this?

Reply:
lol, no, thanks for catching 😅

Reply:
I forgot to push a commit - sorry for having you review this before it was ready.